test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/https/WorkbenchHttpsIntegrationTest.java

```diff
@@ -102,48 +101,66 @@ public void testLoginScreen() throws InterruptedException {
-            url = new URL(url, organizationalUnitRestRequest);
             SSLConnectionSocketFactory sslsf = getSSLConnectionSocketFactory();
-
             CredentialsProvider credentialsProvider = getWorkbenchCredentialsProvider();
             try (CloseableHttpClient httpClient = HttpClients.custom().setSSLSocketFactory(sslsf).setDefaultCredentialsProvider(credentialsProvider).build()) {
                 // Create organizational unit using REST API
-                HttpPost createOrgUnitRequest = new HttpPost(url.toString());
-                createOrgUnitRequest.setHeader("Content-Type", "application/json");
-                createOrgUnitRequest.setEntity(new StringEntity(createOrganizationalUnitJson(ORGANIZATION_UNIT_NAME)));
-                try (CloseableHttpResponse response = httpClient.execute(createOrgUnitRequest)) {
-                    Assertions.assertThat(response.getStatusLine().getStatusCode()).isEqualTo(HttpsURLConnection.HTTP_ACCEPTED);
-                }
+                Assertions.assertThat(createOrganizationalUnit(ORGANIZATION_UNIT_NAME, httpClient)).isTrue();
 
                 // Check that organizational unit exists
-                HttpGet getOrgUnitRequest = new HttpGet(url.toString());
-                getOrgUnitRequest.setHeader("Content-Type", "application/json");
-                try (CloseableHttpResponse response = httpClient.execute(getOrgUnitRequest)) {
-                    Assertions.assertThat(response.getStatusLine().getStatusCode()).isEqualTo(HttpsURLConnection.HTTP_OK);
-                    try (InputStream inputStream = response.getEntity().getContent()) {
-                        String responseContent = IOUtils.toString(inputStream, "UTF-8");
-                        Assertions.assertThat(responseContent).isNotEmpty();
-
-                        Gson gson = new Gson();
-                        JsonArray organizationalUnits = gson.fromJson(responseContent, JsonArray.class);
-                        boolean found = false;
-                        for (JsonElement item : organizationalUnits) {
-                            if (item.isJsonObject()) {
-                                JsonObject organizationalUnit = item.getAsJsonObject();
-                                if (organizationalUnit.get("name").getAsString().equals(ORGANIZATION_UNIT_NAME)) {
-                                    found = true;
-                                }
-                            }
-                        }
-
-                        Assertions.assertThat(found).isTrue();
-                    }
-                }
+                Assertions.assertThat(organizationalUnitExists(ORGANIZATION_UNIT_NAME, httpClient)).isTrue();
+            }
+        } catch (Exception e) {
+            throw new RuntimeException("Unable to connect to workbench REST API", e);
+        }
+    }
+
+    private boolean createOrganizationalUnit(String name, CloseableHttpClient httpClient) {
+        final HttpPost createOrgUnitRequest;
```

Comment:
If the entire try-catch block were its own method, you could've just returned the `createOrgUnitRequest` from within it and have one less needless line of code.